client/components/registration/registration.js

```diff
@@ -24,6 +24,8 @@ import Modal from "../modal";
 import getError from "../../utils/get-error";
 import getLanguageHeaders from "../../utils/get-language-headers";
 import redirectToPayment from "../../utils/redirect-to-payment";
+import InfoModal from "../../utils/modal";
```

Comment:
why are you calling this `InfoModal` here while `Modal` in the status component? I would avoid this because it makes it harder to find where this component is used, one may think these are 2 different things but it looks to me it's the same component named differently, which is not a good idea unless the component is behaving in 2 completely different ways.